code/style.css

```diff
@@ -9,50 +11,141 @@ p {
   background-color: #fff;
 }
 
+.header-styling {
+  color: rgba(10, 11, 13, 0.4);
+  text-shadow: 2px 4px 4px white;
+  text-align: right;
+  background: rgba(10, 11, 13, 0.7);
+  height: 100px;
+  font-size: 70px;
+  padding: 30px;
+}
+
 .cards {
   margin: 50px auto;
   width: 90%;
   height: 600px;
-  background: #ccc;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  border-radius: 20px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.2);
 }
 
 .killer-deck {
   width: 250px;
   height: 400px;
-  background: #000;
+  background: rgba(10, 11, 13, 0.7);
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  box-shadow: 0 4px 8px 0 rgba(27, 107, 29, 0.2),
+    0 6px 20px 0 rgba(221, 217, 217, 0.4);
+  transition: transform 0.2s;
+}
+
+.killer-deck:hover {
+  transform: scale(1.1);
 }
 
 .weapon-deck {
   width: 250px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background: rgba(10, 11, 13, 0.7);
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  box-shadow: 0 4px 8px 0 rgba(27, 107, 29, 0.2),
+    0 6px 20px 0 rgba(221, 217, 217, 0.4);
+  transition: transform 0.2s;
+}
+
+.weapon-deck:hover {
+  transform: scale(1.1);
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
-
+  background: rgba(10, 11, 13, 0.7);
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  box-shadow: 0 4px 8px 0 rgba(27, 107, 29, 0.2),
+    0 6px 20px 0 rgba(221, 217, 217, 0.4);
+  transition: transform 0.2s;
+}
+
+.room-deck:hover {
+  transform: scale(1.1);
 }
 
 button {
   padding: 8px 16px;
-  background: crimson;
+  background: rgba(10, 11, 13, 0.7);
+  color: white;
   border: none;
   font-size: 20px;
-  border-radius: 8px;
+  border-radius: 12px;
   display: block;
   margin: 0 auto;
+  margin-bottom: 20px;
+  box-shadow: 0 4px 8px 0 rgba(27, 107, 29, 0.2),
+    0 6px 20px 0 rgba(221, 217, 217, 0.4);
+  transition-duration: 0.3s;
+}
+
+button:hover {
+  background: #114555;
+  box-shadow: 0 4px 8px 0 rgba(27, 107, 29, 0.2),
+    0 6px 20px 0 rgba(221, 217, 217, 0.4);
+  transform: scale(1.1);
+}
+
+.footer-style {
+  color: white;
+  background: rgba(10, 11, 13, 0.7);
+  height: 50px;
+}
+
+.mystery-text {
+  color: white;
+  text-align: center;
+  margin: 30px;
+}
+
+.loader {
+  position: absolute;
+  font-size: 8px;
+  text-align: center;
+  border-right: 8px solid grey;
+  border-left: 8px solid darkgrey;
+  border-radius: 50%;
+  width: 50px;
+  height: 50px;
+  animation: spin 2s infinite;
+  display: none;
+}
+
+@keyframes spin {
+  0% {
+    transform: rotate(0deg) rotateY(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg) rotateY(360deg);
+  }
+}
+
+.load-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.room-deck .load-wrapper {
+  margin-top: 20px;
+}
+
+.weapon-deck .load-wrapper {
+  margin-top: 20px;
 }
```

Comment:
Overall I think your styling makes sense, it´s readable and you use the correct syntax / properties to adjust the styling according to the needs. I really like the design as well. Good work!

Reply:
Well said!